libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1049,14 +1049,14 @@ bool AP_AHRS_DCM::get_position(struct Location &loc) const
 // return an airspeed estimate if available
 bool AP_AHRS_DCM::airspeed_estimate(float &airspeed_ret) const
 {
-    return airspeed_estimate(_airspeed?_airspeed->get_primary():0, airspeed_ret);
+    return airspeed_estimate(AP::airspeed()?AP::airspeed()->get_primary():0, airspeed_ret);
 }
 
 // return an airspeed estimate from a specific airspeed sensor instance if available
 bool AP_AHRS_DCM::airspeed_estimate(uint8_t airspeed_index, float &airspeed_ret) const
 {
     if (airspeed_sensor_enabled(airspeed_index)) {
-        airspeed_ret = _airspeed->get_airspeed(airspeed_index);
+        airspeed_ret = AP::airspeed()->get_airspeed(airspeed_index);
```

Comment:
AP::airspeed() needs nullptr check

Reply:
This is checked in `airspeed_sensor_enabled`.

I'm looking at refactoring a whole bunch of this stuff and would prefer to not make too many changes.  Note that `_airspeed` here is *already* a pointer, so the nullptr check better work already!